src/components/Timeline/TimelineBrowser.js

```diff
@@ -13,52 +13,64 @@ import Svg, {
 import { timelineIntervalsSelector } from '../../redux/selectors';
 
 const BAR_COLOR = '#666';
+const BAR_COLOR_1SD = '#c33';
+const BAR_COLOR_2SD = '#f00';
```

Comment:
the standard deviation reds are pretty subtle, may want to make it more different